cmd/kubeadm/app/cmd/token.go

```diff
@@ -94,10 +93,10 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 	var description string
 	createCmd := &cobra.Command{
 		Use:   "create [token]",
-		Short: "Create bootstrap tokens on the server.",
+		Short: "Create Bootstrap Tokens on the server.",
```

Comment:
Nit: I like being consistent, but I'd rather go with lowercase.

Reply:
ok, making this lowercase everywhere.